src/commons/util/httpclient.rs

```diff
@@ -58,6 +58,24 @@ fn report_delete(uri: &str, content_type: Option<&str>, token: Option<&Token>) {
     }
 }
 
+/// Gets the Bearer token from the request header, if present.
+pub fn get_bearer_token(request: &hyper::Request<hyper::Body>) -> Option<Token> {
+    if let Some(header) = request.headers().get("Authorization") {
```

Comment:
This was not caused by you but I don't like the hard-coded string header name which could easily be broken by a typo, we should instead use https://docs.rs/hyper/0.14.12/hyper/header/constant.AUTHORIZATION.html. I'm not sure if it's really better but this could also be written [in more functional style like this](https://gist.github.com/rust-play/e2013acd78fd6b3e355febe387b9ecc1).

Reply:
I agree.. now fixed.